blockchain/vm/evm.go

```diff
@@ -452,6 +452,7 @@ func (evm *EVM) create(caller types.ContractRef, codeAndHash *codeAndHash, gas u
 	// TODO-Klaytn-Accounts: for now, smart contract accounts cannot withdraw KLAYs via ValueTransfer
 	//   because the account key is set to AccountKeyFail by default.
 	//   Need to make a decision of the key type.
+	codeFormat.SetIstanbulHFField(evm.chainRules.IsIstanbul)
```

Comment:
There are other places `CreateSmartContractAccountWithKey()` is called. Why don't you add `codeFormat.SetIstanbulHFField()` inside of `CreateSmartContractAccountWithKey()` ?